vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,31 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    if (env.STAGE_NAME.contains('run_test.sh')) {
+      result['junit_files'] = 'test_results/*.xml'
+      result['artifacts'] = ['run_test.sh/*', 'vm_test/**']
+      result['with_valgrind'] = ''
```

Comment:
Key should just not be present.

Reply:
how should this be managed? I thought we wanted to know this information (if run with valgrind or not) from the stage name

Reply:
If the 'with_valgrind' key is not present then it means that valgrind tests are not wanted, so the default is no key.